packages/mariadb-10.11-mroonga/debian/rules

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/make -f
+# -*- makefile-gmake -*-
+#
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+# This has to be exported to make some magic below work.
+export DH_OPTIONS
+
+export MARIADB_VERSION_FULL := $(shell apt-cache show mariadb-server | grep Version | head -n 1 | awk '{print $$2}' | cut -d: -f2)
+export MARIADB_VERSION := $(shell apt-cache show mariadb-server | grep Version | head -n 1 | awk '{print $$2}' | cut -d: -f2 | awk -F '-' '{print $$1}')
+
+%:
+	dh $@
```

Comment:
Ah, it may be better that we specify `--buildsystem` here instead of `dh_auto_configure`:
 
```suggestion
	dh $@ --buildsystem=cmake+ninja
```

If we specify `--buildsystem` for `dh_auto_configure`, we also need to specify it to `dh_auto_install`.

See also: https://www.clear-code.com/blog/2018/5/25.html